src/main/plugins/org.dita.pdf2/xsl/fo/tables.xsl

```diff
@@ -962,61 +989,82 @@ See the accompanying LICENSE file for applicable license.
          default headings for the table. By default, the existing sthead
          element is used when specified. -->
     <xsl:template match="*[contains(@class, ' topic/simpletable ')]" mode="dita2xslfo:simpletable-heading">
-        <xsl:param name="number-cells" as="xs:integer">
+        <xsl:param name="number-cells" as="xs:integer" tunnel="yes">
             <xsl:apply-templates select="*[1]" mode="count-max-simpletable-cells"/>
         </xsl:param>
         <xsl:apply-templates select="*[contains(@class, ' topic/sthead ')]">
-            <xsl:with-param name="number-cells" select="$number-cells"/>
+            <xsl:with-param name="number-cells" select="$number-cells" tunnel="yes"/>
         </xsl:apply-templates>
     </xsl:template>
 
     <xsl:template match="*[contains(@class, ' topic/sthead ')]">
-      <xsl:param name="number-cells" as="xs:integer">
+      <xsl:param name="number-cells" as="xs:integer" tunnel="yes">
             <xsl:apply-templates select="../*[1]" mode="count-max-simpletable-cells"/>
         </xsl:param>
         <fo:table-header xsl:use-attribute-sets="sthead">
             <xsl:call-template name="commonattributes"/>
             <fo:table-row xsl:use-attribute-sets="sthead__row">
                 <xsl:apply-templates select="*[contains(@class, ' topic/stentry ')]"/>
-                <xsl:variable name="row-cell-count" select="count(*[contains(@class, ' topic/stentry ')])" as="xs:integer"/>
+                <!--
```

Comment:
Was this commented section (and the one below) left in intentionally?

Reply:
Yeah, I don't want to throw away the code, if we end up wanting to use it.